ui-admin/src/study/participants/enrolleeView/EnrolleeProfile.tsx

```diff
@@ -1,82 +1,179 @@
+    setProfile((oldVal: Profile) => {
+      return {
+        ...oldVal,
+        mailingAddress
+      }
+    })
+  }
+
+  const onStringFieldChange = (field: string, value: string) => {
+    setProfile((oldVal: Profile) => {
+      return {
+        ...oldVal,
+        [field]: value
+      }
+    })
+  }
+
+  const onBooleanFieldChange = (field: string, value: boolean) => {
+    setProfile((oldVal: Profile) => {
+      return {
+        ...oldVal,
+        [field]: value
+      }
+    })
+  }
+
   return <div>
+    <input type={'checkbox'}
+      checked={editMode}
+      onChange={(e: React.ChangeEvent<HTMLInputElement>) => setEditMode(e.target.checked)}
+    />
     <form className="mb-3">
       <div>
         <label className="form-label">
-          Given name: <input className="form-control" type="text" readOnly={true} value={enrollee.profile.givenName}/>
+          Given name: <input className="form-control" type="text" disabled={!editMode}
+            value={profile.givenName}
+            onChange={e => onStringFieldChange('givenName', e.target.value)}/>
         </label>
         <label className="form-label ms-2">
-          Family name: <input className="form-control" type="text" readOnly={true} value={enrollee.profile.familyName}/>
+          Family name: <input className="form-control" type="text" disabled={!editMode}
+            value={profile.familyName}
+            onChange={e => onStringFieldChange('familyName', e.target.value)}/>
         </label>
       </div>
       <div className="mb-3">
         <label className="form-label">
           Contact email:
-          <input className="form-control" type="text" readOnly={true} value={enrollee.profile.contactEmail}/>
+          <input className="form-control" type="text" disabled={!editMode} value={profile.contactEmail}
+            onChange={e => onStringFieldChange('contactEmail', e.target.value)}/>
         </label>
         <label className="form-label ms-3">
           Do not email:
-          <input type="checkbox" className="ms-1" disabled={true} checked={enrollee.profile.doNotEmail}/>
+          <input type="checkbox" className="ms-1" disabled={!editMode} checked={profile.doNotEmail}
+            onChange={e => onBooleanFieldChange('doNotEmail', e.target.checked)}/>
         </label>
         <label className="form-label ms-3">
           Do not solicit:
-          <input type="checkbox" className="ms-1" disabled={true} checked={enrollee.profile.doNotEmailSolicit}/>
+          <input type="checkbox" className="ms-1" disabled={!editMode} checked={profile.doNotEmailSolicit}
+            onChange={e => onBooleanFieldChange('doNotEmailSolicit', e.target.checked)}/>
         </label>
       </div>
       <div className="mb-3">
         <label className="form-label">
-          Birthdate:
-          <input className="form-control" type="text"
-            readOnly={true} value={dateToDefaultString(enrollee.profile.birthDate)}/>
+          Birthdate (MM/DD/YYYY):
```

Comment:
We should use a date-picker widget for this, no sense rolling our own and forcing HeartHive to use US-formatted input